docs/content/preview/manage/ysql-major-upgrade.md

```diff
@@ -0,0 +1,190 @@
+---
+title: YSQL major upgrade
+headerTitle: YSQL major upgrade
+linkTitle: YSQL major upgrade
+description: Upgrade YugabyteDB to PostgreSQL 15
+headcontent: Upgrade YugabyteDB to a version that supports PG 15
+menu:
+  preview:
+    identifier: ysql-major-upgrade
+    parent: manage-upgrade-deployment
+    weight: 706
+type: docs
+---
+
+Upgrading YugabyteDB from a version based on PostgreSQL 11 (all versions prior to v2.25) to a version based on PostgreSQL 15 (v2.25.1 or later) requires some additional steps.
+
```

Comment:
Lets link back to upgrade-deployment.md for normal upgrades within pg15, or pg11